### ISSUE TITLE
CopyMeta module

### ISSUE THREAD

**Issue by daveaglick** [statiqdev/Statiq#207]
Copies metadata from one key to another. Consider additional constructor overload that performs string formatting before setting the destination value.

**Comment by deanebarker** [comment]
I'll get this one before the weekend.

**Comment by deanebarker** [comment]
Couldn't sleep. Pull request #209.

This cleaned up my config file enormously.  Consider before and after:

```
// Before
Meta("Url", $"/posts/{@doc.String("SourceFileBase")}/")

// After
CopyMeta("SourceFileBase", "Url").WithFormat("/posts/{0}/")

// Or with format as an optional constructor argument...
CopyMeta("SourceFileBase", "Url", "/posts/{0}/")
```

Toying with the idea of an `AsType` method.  Like this:

```
CopyMeta("StringDate", "TypedDate").AsType(typeof(DateTime))
```

Not sure about that yet, but something for the future perhaps.

**Comment by daveaglick** [comment]
Very nice! Closed by #209
